include/dd/ComplexTable.hpp

```diff
@@ -371,50 +362,98 @@ namespace dd {
         }
 
         [[nodiscard]] fp hitRatio() const { return static_cast<fp>(hits) / lookups; }
+
         [[nodiscard]] fp colRatio() const { return static_cast<fp>(collisions) / lookups; }
 
         std::ostream& printStatistics(std::ostream& os = std::cout) {
-            os << "hits: " << hits << ", collisions: " << collisions << ", looks: " << lookups << ", hitRatio: " << hitRatio() << ", colRatio: " << colRatio() << ", gc calls: " << gcCalls << ", gc runs: " << gcRuns << "\n";
+            // clang-format off
+            os << "hits: " << hits
+               << ", collisions: " << collisions
+               << ", looks: " << lookups
+               << ", breaks: " << breaks
+               << ", neighbors: " << neighbors
+               << ", hitRatio: " << hitRatio()
+               << ", colRatio: " << colRatio()
+               << ", gc calls: " << gcCalls
+               << ", gc runs: " << gcRuns
+               << "\n";
+            //clang-format on
```

Comment:
```suggestion
            // clang-format on
```
I guess clang format is not turned back on again when the whitespace is missing.